### PR TITLE
test

### PR DESCRIPTION
**Pull request by olexey81**
test

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -14,17 +14,6 @@ static void Main(string[] args)
             Console.WriteLine("Hello, World!");
             Console.WriteLine("Hello, World!");
             Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
```
Comment (O-Kryvchykov) [inline Program.cs:20]: test 1

File: Program.cs [hunk 2]
```diff
@@ -14,17 +14,6 @@ static void Main(string[] args)
...
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
```
Comment (O-Kryvchykov) [inline Program.cs:25]: test 2

File: Program.cs [hunk 3]
```diff
@@ -14,17 +14,6 @@ static void Main(string[] args)
             Console.WriteLine("Hello, World!");
             Console.WriteLine("Hello, World!");
             Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
```
Comment (O-Kryvchykov) [inline Program.cs:19]: test 1

File: Program.cs [hunk 4]
```diff
@@ -14,17 +14,6 @@ static void Main(string[] args)
...
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
```
Comment (O-Kryvchykov) [inline Program.cs:24]: test 2

**Review by O-Kryvchykov — COMMENTED** [review]
trst all

File: Program.cs (hunk 3 above)
Comment (olexey81) [reply Program.cs:19]: retest 2

File: Program.cs (hunk 4 above)
Comment (olexey81) [reply Program.cs:24]: retest 3

**Comment by olexey81** [comment]
done

File: Program.cs (hunk 3 above)
Comment (olexey81) [reply Program.cs:19]: qwe

File: Program.cs (hunk 4 above)
Comment (olexey81) [reply Program.cs:24]: reytery

File: Program.cs (hunk 3 above)
Comment (olexey81) [reply Program.cs:19]: vjgkghl

File: Program.cs [hunk 5]
```diff
@@ -14,17 +14,6 @@ static void Main(string[] args)
             Console.WriteLine("Hello, World!");
             Console.WriteLine("Hello, World!");
             Console.WriteLine("Hello, World!");
```
Comment (O-Kryvchykov) [inline Program.cs:16]: dfhfl,jk/;k;hgmfh

File: Program.cs [hunk 6]
```diff
@@ -14,17 +14,6 @@ static void Main(string[] args)
...
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
-            Console.WriteLine("Hello, World!");
```
Comment (O-Kryvchykov) [inline Program.cs:23]: dsgsdghsdghsgsd

File: Program.cs (hunk 5 above)
Comment (olexey81) [reply Program.cs:16]: sdvfsbnfgmf

File: Program.cs (hunk 6 above)
Comment (olexey81) [reply Program.cs:23]: 25423634623632

**Review by olexey81 — COMMENTED** [review]
jskdhgskhifh23y7r52893u523ghnd,jnhgdfbhdfb

**Pull request merged**